docker/build_docker_sagemaker.sh

```diff
@@ -39,7 +39,7 @@ cp -r "${GSF_HOME}/sagemaker" code/graphstorm/sagemaker
 cp -r "${GSF_HOME}/docker/sagemaker/build_artifacts" build_artifacts
 
 # Build OSS docker for EC2 instances that an pull ECR docker images
-DOCKER_FULLNAME="${IMAGE_NAME}:${TAG}"
+DOCKER_FULLNAME="${IMAGE_NAME}:${TAG}-${IMAGE_TYPE}"
```

Comment:
DEVICE_TYPE?

Reply:
We're using IMAGE_TYPE right now see https://github.com/awslabs/graphstorm/blob/b0a8c7750896c8aa0866be2318c25fabbf850b48/docker/build_docker_sagemaker.sh#L13-L18

Agree that DEVICE_TYPE is a better name, I can change it to that if you think we should.

Reply:
Let's use DEVICE_TYPE